tests/main/upgrade-from-2.15/task.yaml

```diff
@@ -3,10 +3,18 @@ summary: Ensure upgrades from snapd 2.15 work
 systems: [ubuntu-16.04-64]
 
 prepare: |
-    dpkg --purge snapd
+    #shellcheck source=tests/lib/pkgdb.sh
```

Comment:
Is this relevant for the test? This test is specific to Ubuntu anyway.

Reply:
It is done like this becasuse it is specific for ubuntu xenial